src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/XPlatUtility.cs

```diff
@@ -67,8 +67,7 @@ public static void ConfigureProtocol()
         public static void SetUserAgent()
         {
 #if IS_CORECLR
-            UserAgent.SetUserAgentString(new UserAgentStringBuilder("NuGet xplat")
-                .WithOSDescription(RuntimeInformation.OSDescription));
+            UserAgent.SetUserAgentString(new UserAgentStringBuilder("NuGet xplat"));
 #else
             UserAgent.SetUserAgentString(new UserAgentStringBuilder("NuGet xplat"));
 #endif
```

Comment:
It looks like the compiler directive isn't needed anymore. In both cases I think it's doing the same thing now.